apollo-test/src/main/java/com/spotify/apollo/test/unit/matchers/request/HasNoQueryParametersMatcher.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * -\-\-
+ * Spotify Apollo Testing Helpers
+ * --
+ * Copyright (C) 2013 - 2017 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.apollo.test.unit.matchers.request;
```

Comment:
I was asking because all the other matchers are in `com.spotify.apollo.test.unit` and if we kept this class and the other "accessory" matchers in the same folder they could be package private? But it's just a suggestion.. I'll leave it up to you.